core/src/test/java/com/microsoft/applicationinsights/telemetry/ExceptionTelemetryTest.java

```diff
@@ -34,17 +34,6 @@
 
 public final class ExceptionTelemetryTest {
 
-    @Test
-    public void testSetException() {
-        NullPointerException exception = new NullPointerException("mock");
-        ExceptionTelemetry exceptionTelemetry = new ExceptionTelemetry(exception);
-
-        NullPointerException exception1 = new NullPointerException("mock");
-        exceptionTelemetry.setException(exception1);
```

Comment:
It seems that you have removed this setter in ExceptionTelemetry class. It was a public method and therefore it means it could be a breaking change. I would like to avoid any breaking change here.

Reply:
I've reverted this change.